index.js

```diff
@@ -63,6 +63,33 @@ async function pushBranches(sg, branches, forcePush, remote = DEFAULT_REMOTE) {
   console.log('All changes pushed ' + emoji.get('white_check_mark'));
 }
 
+async function checkoutNewBranch(sg, newBranch){
+  try {
+    await sg.raw(['checkout', '-b', newBranch]);
+  } catch (e) {
+    console.log(`${newBranch} is an existing branch... Checking out`)
+    await sg.raw(['checkout', newBranch]);
+  }
```

Comment:
This gives me `fatal: A branch named 'new-branch-name' already exists.`  Not sure how to avoid it.